src/renderer/components/Validator/ValidatorSimple.tsx

```diff
@@ -19,24 +22,45 @@ export interface IValidatorSimpleProps {
 
 export const ValidatorSimple: React.FunctionComponent<IValidatorSimpleProps> = (
     props: IValidatorSimpleProps) => {
+    const dispatch = useDispatch();
+
+    useEffect(() => {
+        // Initially load beacon nodes
+        dispatch(loadValidatorBeaconNodes(props.publicKey));
+
+        // Periodically refresh status with latest height
+        const timeoutID = setInterval(() => {
+            dispatch(loadValidatorBeaconNodes(props.publicKey));
+        }, 5000);
```

Comment:
Can you add method in eth2ApiClient onNewHead. Some bn will have streaming api for that, as backup you can add this interval invoking :)

Reply:
What should that method do?

Reply:
it subscription api it will execute param everytime bn processes block